### PR TITLE
General topics

### PR DESCRIPTION
**Pull request by AndrePanin**
Added general topics articles.

### CODE REVIEW

File: docs/general/dApps.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 2
+sidebar_label: 'dApps'
```
Comment (xrdavies) [inline docs/general/dApps.md:3]: Is it better to use `dApp` instead of `dApps` to be more precise?

File: docs/general/dao.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 6
+sidebar_label: 'DAOs'
```
Comment (xrdavies) [inline docs/general/dao.md:3]: Maybe `DAO` instead of `DAOs` here.

File: docs/general/nft.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 5
+sidebar_label: 'NFTs'
```
Comment (xrdavies) [inline docs/general/nft.md:3]: The same here. Maybe `NFT` is better than `NFTs`.

File: docs/general/dApps.md (hunk 1 above)
Comment (AndrePanin) [reply docs/general/dApps.md]: ok, updated

File: docs/general/dao.md (hunk 2 above)
Comment (AndrePanin) [reply docs/general/dao.md]: ok, updated

File: docs/general/nft.md (hunk 3 above)
Comment (AndrePanin) [reply docs/general/nft.md]: ok, updated

**Pull request merged**